contribute.html

```diff
@@ -14,37 +14,65 @@
 
 	</head>
 		<body>
-	  			<header>
-						<div class="container">
-	  				<div class="logo"><h1>Podcastr</h1>
-								<h2>For Podcast Addicts.<br>
-								By Podcast Addicts.</h2>
+			<header>
+				<div class="blackback">
+					<div class="container">
+							<div class="logo">
+									<h1>Podcastr</h1>
+									<h2>Not your grandpa's radio.</h2>
+								</div>
+								<div class="search">
+										<form>
+											<input type="text" id="findapodcast" placeholder="Find a podcast...">
+											<input type="submit" value="Search" id="submit-btn">
+										</form>
+									</div>
+								<nav>
+									<ul>
+										<li><a href="file:///Users/Josh/Desktop/GA%20FEWD/finalproject/index.html">Home</a></li>
```

Comment:
Don't add file:///....
Make your file relative. like ./index.html. Otherwise, if you were to ever move it to a hosting service your site would break.